navigator/backend/src/main/scala/com/digitalasset/navigator/data/DatabaseActions.scala

```diff
@@ -95,11 +96,19 @@ class DatabaseActions {
     } yield SqlQueryResult(colNames, data.map(_.map(Option(_).map(_.toString).getOrElse("null"))))
   }
 
-  def runQuery(query: String): Try[SqlQueryResult] = {
+  /** Returns the given value, logging failures */
+  private def logErrors[T](result: Try[T]): Try[T] =
+    result.recoverWith {
```

Comment:
I think this is better expressed as:
```
result.failed.foreach { case NonFatal(t) => logger.error(....) }
result
```

The `recoverWith` indicates that some recovery checking is done, when it's actually not.